meshroom/core/node.py

```diff
@@ -556,6 +556,15 @@ def getColor(self):
             return self.internalAttribute("color").value.strip()
         return ""
 
+    def setColor(self, color: str):
```

Comment:
With regards to what this PR is addressing, adding the setter for this property is not needed.